OrbitGl/App.h

```diff
@@ -303,18 +303,18 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
 
   void CrashOrbitService(orbit_grpc_protos::CrashOrbitServiceRequest_CrashType crash_type);
 
+  void SetProcess(ProcessData* process) {
+    CHECK(process != nullptr);
+    process_ = process;
+  }
   [[nodiscard]] DataView* GetOrCreateDataView(DataViewType type) override;
   [[nodiscard]] DataView* GetOrCreateSelectionCallstackDataView();
 
   [[nodiscard]] ProcessManager* GetProcessManager() { return process_manager_.get(); }
   [[nodiscard]] ThreadPool* GetThreadPool() { return thread_pool_.get(); }
   [[nodiscard]] MainThreadExecutor* GetMainThreadExecutor() { return main_thread_executor_.get(); }
-  [[nodiscard]] ProcessData* GetMutableSelectedProcess() const {
-    return data_manager_->mutable_selected_process();
-  }
-  [[nodiscard]] const ProcessData* GetSelectedProcess() const {
-    return data_manager_->selected_process();
-  }
+  [[nodiscard]] ProcessData* GetMutableSelectedProcess() const { return process_; }
+  [[nodiscard]] const ProcessData* GetSelectedProcess() const { return process_; }
```

Comment:
It's confusing to call the getter `Get**Selected**Process`, but the setter `SetProcess`. Sound like these can be different processes. I suggest to go for "TargetProcess", or simply "Process" in both cases.

By the way: Would it make sense to have the target process live in the Process manager, and remove the whole "Get/SetProcess" and "process_" from App? Currently the process manager does not really manage anything, it only lists things. But this should be done in a subsequent PR if we want this.

Reply:
This happened because it was called GetSelectedProcess for a while now. I agree that just GetProcess (and GetMutableProcess) is better.

Reply:
Done, changed it to "TargetProcess" to communicate clearly that this is the currently profiled process.